sound/soc/sof/topology.c

```diff
@@ -1918,6 +1918,8 @@ static int sof_manifest(struct snd_soc_component *scomp, int index,
 
 /* vendor specific kcontrol handlers available for binding */
 static const struct snd_soc_tplg_kcontrol_ops sof_io_ops[] = {
+	{SND_SOC_TPLG_CTL_VOLSW, snd_sof_volume_get, snd_sof_volume_put,
+		snd_sof_volume_info},
```

Comment:
Should this info be appended to SOF_TPLG_KCTL_VOL_ID ?

Reply:
yes, I would do this.  I did this according to id, the smallest in the head